v2/api/dataprotection/v1api20230101/structure.txt

```diff
@@ -0,0 +1,1272 @@
+│   │   │           │               │       └── WeeksOfTheMonth: Enum (5 values)[]
+│   │   │           │               │           ├── "First"
+│   │   │           │               │           ├── "Fourth"
+│   │   │           │               │           ├── "Last"
+│   │   │           │               │           ├── "Second"
+│   │   │           │               │           └── "Third"
+│   │   │           │               ├── IsDefault: *bool
+│   │   │           │               ├── TagInfo: *Object (1 property)
+│   │   │           │               │   └── TagName: *string
+│   │   │           │               └── TaggingPriority: *int
+│   │   │           └── AzureRetention: *Object (4 properties)
+│   │   │               ├── IsDefault: *bool
+│   │   │               ├── Lifecycles: Object (3 properties)[]
+│   │   │               │   ├── DeleteAfter: *Object (1 property)
+│   │   │               │   │   └── AbsoluteDeleteOption: *Object (2 properties)
+│   │   │               │   │       ├── Duration: *string
+│   │   │               │   │       └── ObjectType: *Enum (1 value)
+│   │   │               │   │           └── "AbsoluteDeleteOption"
+│   │   │               │   ├── SourceDataStore: *Object (2 properties)
+│   │   │               │   │   ├── DataStoreType: *Enum (3 values)
+│   │   │               │   │   │   ├── "ArchiveStore"
+│   │   │               │   │   │   ├── "OperationalStore"
+│   │   │               │   │   │   └── "VaultStore"
+│   │   │               │   │   └── ObjectType: *string
+│   │   │               │   └── TargetDataStoreCopySettings: Object (2 properties)[]
+│   │   │               │       ├── CopyAfter: *Object (3 properties)
+│   │   │               │       │   ├── CopyOnExpiry: *Object (1 property)
+│   │   │               │       │   │   └── ObjectType: *Enum (1 value)
+│   │   │               │       │   │       └── "CopyOnExpiryOption"
+│   │   │               │       │   ├── CustomCopy: *Object (2 properties)
+│   │   │               │       │   │   ├── Duration: *string
+│   │   │               │       │   │   └── ObjectType: *Enum (1 value)
+│   │   │               │       │   │       └── "CustomCopyOption"
+│   │   │               │       │   └── ImmediateCopy: *Object (1 property)
+│   │   │               │       │       └── ObjectType: *Enum (1 value)
+│   │   │               │       │           └── "ImmediateCopyOption"
+│   │   │               │       └── DataStore: *Object (2 properties)
+│   │   │               │           ├── DataStoreType: *Enum (3 values)
+│   │   │               │           │   ├── "ArchiveStore"
+│   │   │               │           │   ├── "OperationalStore"
+│   │   │               │           │   └── "VaultStore"
+│   │   │               │           └── ObjectType: *string
+│   │   │               ├── Name: *string
+│   │   │               └── ObjectType: *Enum (1 value)
+│   │   │                   └── "AzureRetentionRule"
+│   │   └── SystemData: *Object (6 properties)
```

Comment:
Should this be marked readonly, like you did for the other resource? (see your existing edits to azure-arm.yaml)

Reply:
SystemData property is updated to read-only for this too.